test/fakedata_generation.py

```diff
@@ -57,37 +57,6 @@ def clean_dir(root, *keep):
             dir_util.remove_tree(file_or_dir)
 
 
-@contextlib.contextmanager
```

Comment:
Can we keep this for now just to be safe, as we might have internal users relying on this? You can add a comment saying that this isn't used in torchvision anymore